### PR TITLE
terms of use page TA

### PR DESCRIPTION
**Pull request by KietasRiestutelis**

### CODE REVIEW

File: src/step-definitions/termsOfUse.steps.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Then(/^date under Terms Of Use title is correct$/, async () => {
+    await expectText(TermsOfUse.termsOfUseMainContentDate, Data.termsOfUseMainContentDateText);
+});
+
+Then(/^Terms Of Use content text is correct$/, async () => {
+    await expectToExist(TermsOfUse.termsOfUseMainContentElem);
+    const expectedContent = readFileSync(join(__dirname, '../data/termsOfUseContent-text.txt'), 'utf-8');
+    await expectText(TermsOfUse.termsOfUseMainContentElem, expectedContent);
+    // console.log("AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA");
```
Comment (Deimokas) [inline src/step-definitions/termsOfUse.steps.ts]: commented code should not be left

File: src/data/global-data.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ const yargs = require('yargs').argv;
 export const LaunchHeadlessMode = true; // If `false`, a browser will auto open (useful to monitor visually instead of through terminal)
 
 export const Specifications = [
-  './src/features/*.feature',
```
Comment (Deimokas) [inline src/data/global-data.ts]: this should be removed and left to default

File: src/page-objects/quizzard.page.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ import { Page } from './page';
 
 export default class quizzard extends Page {
   open() {
-    return super.open(urls.quizLinks.Vision);
+    return super.open(urls.quizLinks.vision);
```
Comment (Deimokas) [inline src/page-objects/quizzard.page.ts:11]: Why is this change in this branch?

File: src/page-objects/quizzard.page.ts [hunk 4]
```diff
@@ -28,7 +28,7 @@ export default class quizzard extends Page {
   }
 
   async validateVisionUrl() {
-    await expectURL(urls.quizLinks.Vision);
```
Comment (Deimokas) [inline src/page-objects/quizzard.page.ts:31]: This looks like changes from different branch which were left here unintendedly

File: src/page-objects/termsOfUse.page.ts [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      }
+
+    get termsOfUseMainContentDate(){
+        return getElement('[ta-id="terms-of-use_266143761"]');
+    } 
+
+    get termsOfUseMainContentElem () {
+        return getElement('[ta-id="terms-of-use_167470060"]');
+    }
+
```
Comment (Deimokas) [inline src/page-objects/termsOfUse.page.ts]: unnesesary changes

**Review by Deimokas — CHANGES_REQUESTED** [review]

**Review by alexevic — COMMENTED** [review]
I agree with @Deimokas notes

File: src/data/global-data.ts [hunk 6]
```diff
@@ -40,4 +40,5 @@ export const ContextPages = {
   
   Faq: ['faq'],
   howItWorks: ['howItWorks'],
+  termsOfUse: ['termsOfUse'],
```
Comment (alexevic) [inline src/data/global-data.ts:43]: Needs to be written in pascal case, like another ContextPages

File: src/step-definitions/termsOfUse.steps.ts [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    await expectAttributeToContain(await TermsOfUse.termsOfUseMainContentIcon, "xlink:href", "/brighter-assets/icons/green_icons.svg#Clipboard-Green");
+});
+
+Then(/^date under Terms Of Use title is correct$/, async () => {
+    await expectText(TermsOfUse.termsOfUseMainContentDate, Data.termsOfUseMainContentDateText);
+});
+
+Then(/^Terms Of Use content text is correct$/, async () => {
+    await expectToExist(TermsOfUse.termsOfUseMainContentElem);
+    const expectedContent = readFileSync(join(__dirname, '../data/termsOfUseContent-text.txt'), 'utf-8');
```
Comment (alexevic) [inline src/step-definitions/termsOfUse.steps.ts:31]: termsOfUseContent-text.txt maybe should be changed to termsOfUseContentText.txt (to match camel case to all file naming)

**Review by alexevic — CHANGES_REQUESTED** [review]
Few small naming changes needed

File: src/step-definitions/termsOfUse.steps.ts (hunk 7 above)
Comment (KietasRiestutelis) [reply src/step-definitions/termsOfUse.steps.ts:31]: These changes will be made in another branch

File: src/data/global-data.ts (hunk 6 above)
Comment (KietasRiestutelis) [reply src/data/global-data.ts:43]: These changes will be made in another branch

**Pull request merged**